code-react/src/views/dashboard/app/user-profile-edit.js

```diff
@@ -44,206 +114,299 @@ const UserProfileEdit =() =>{
-                                                  <Form.Select defaultValue="USA"  className="form-select" aria-label="Default select example 3">
-                                                  <option>Caneda</option>
-                                                  <option>Noida</option>
-                                                  <option>USA</option>
-                                                  <option>India</option>
-                                                  <option>Africa</option>
-                                                  </Form.Select>
-                                              </Form.Group>
-                                              <Form.Group className="form-group col-sm-6">
-                                                  <Form.Label className="form-label">State:</Form.Label>
-                                                  <Form.Select defaultValue="Georgia" className="form-select" aria-label="Default select example 4">
-                                                      <option>California</option>
-                                                      <option>Florida</option>
-                                                      <option>Georgia</option>
-                                                      <option>Connecticut</option>
-                                                      <option>Louisiana</option>
-                                                  </Form.Select>
-                                              </Form.Group>
-                                              <Form.Group className="form-group col-sm-12">
-                                                  <Form.Label className="form-label">Address:</Form.Label>
-                                                  <Form.Control as="textarea" className="form-control" rows={5} style={{lineHeight: "22px"}} placeholder=" 37 Cardinal Lane
-                                                  Petersburg, VA 23803
-                                                  United States of America
-                                                  Zip Code: 85001">
-                                                  </Form.Control>
-                                              </Form.Group>
-                                          </Row>
-                                          <Button type="submit" className="btn btn-primary me-2">Submit</Button>
-                                          <Button type="reset" className="btn bg-soft-danger">Cancel</Button>
-                                      </Form>
-                                  </Card.Body>
-                              </Card>
-                          </Tab.Pane>
-                          <Tab.Pane eventKey="second" className="fade show">
+                          <UserInfo userInput = {userInput} userInfo={userInfo} onUserInput={setUserInput} updateUser={updateUser} refetch = {refetch}/>

```

Comment:
Suggestion if you have time for<UserInfo\>, <ChangePassword\> and <FamilyMember> and ManagConctact 

Its better to make them general components and take out Tab.pane from them 
It make code more readable and makes  each component  reusable

Reply:
Done created 5 more files for different components.
Indeed better